### PR TITLE
README: proposal to add a "reviewers' guideline" 

### PR DESCRIPTION
**Pull request by davcri**
This is only a proposal, please review it and discuss on what we should add/remove/improve !

Things that should be still addressed:

- How we treat PRs that update a stub article appending new content to the stub template

### CODE REVIEW

**Comment by tue96117** [comment]
#freeshirt

**Comment by anujpatle50** [comment]
#freeshirt

**Comment by emberu** [comment]
**

**Comment by davcri** [comment]
https://github.com/freeCodeCamp/guides/pull/2976/commits/bbfbb598c69b7a24700b465984ceac820507b3d4 : updated the guide with suggestions from @erictleung (from [this issue](https://github.com/freeCodeCamp/guides/issues/2231#issuecomment-338454201)) and @Ethan-Arrowood (from the Gitter channel)

File: README.md [hunk 1]
```diff
@@ -208,6 +210,80 @@ Also, there's a community of support from a whole team of contributors, whom you
...
+
+
+### Conflicting and duplicate PRs
+To find duplicates PRs: 
+
+1. Sort PR from the oldest
+1. Search for PRs with similar content
+1. Merge from the oldest to the newest
+
+There will be probably merge conflicts; if it is the case, notify the contributor with this template text:
```
Comment (HKuz) [inline README.md]: Nit: "will be probably" => "will probably be"

File: README.md [hunk 2]
```diff
@@ -208,6 +210,80 @@ Also, there's a community of support from a whole team of contributors, whom you
...
+  
+#### Template text
+> We apologize for the inconvenience; however, while your PR was in the review queue someone else proposed the same file and their contribution was merged. As a result we need to resolve merge conflicts in order to merge your changes. If you're unsure about this process feel free to reach out in the contributor Gitter channel or comment below. We recommend you review the existing file and propose how you could incorporate your own ideas while maintaining the other contributors work. We will be closing this PR for now, but if you still want to see your changes added let us know and we can open it for additional commits.
```
Comment (HKuz) [inline README.md]: Let's add an inline link to the contributor chat room: `in the [Contributor Gitter channel](https://gitter.im/FreeCodeCamp/Contributors)...`. Also "while maintaining the other contributors work" => "while maintaining the other contributor's work"

File: README.md [hunk 3]
```diff
@@ -208,6 +210,80 @@ Also, there's a community of support from a whole team of contributors, whom you
...
+Thank you for contributing! Keep up the awesome work 
+
+### Requesting changes
+
+If a pull request is not perfect we can:
+- request changes to the contributor and adding the *change requested* label (see below)
+- fix minor issues by ourselves
+
+#### Adding Labels
+- **content** is for pull requests that modify the content of articles on the guide (e.g.: new articles or updating articles)
```
Comment (HKuz) [inline README.md]: Another nit: "change requested" => "changes requested" (plural 😜 - same comment for the other two examples below)

**Comment by davcri** [comment]
I pushed a new commit. @HKuz thank you very much for the review. There were a lot of errors, I absolutely need to improve my English 😝 

However: should we put this "_reviewers' guide"_ to the README ? New contributors could be shocked by the wall of text.
What about putting it in a CONTRIBUTING.md file ?

**Comment by HKuz** [comment]
@davcri - no worries - I couldn't write that in another language! Yeah, there has been talk of creating a CONTRIBUTING.md file. I think these changes would be better suited there, and we keep the README as-is. If you want to create that file (root directory is fine), then copy the "How to contribute" and "How to run the Guide locally on your computer" from the README into it (keep those in the README as well) along with your changes here, we can work off that?

**Comment by davcri** [comment]
I moved some text from the README to the CONTRIBUTING file. 

@HKuz can you add here some relevant guideline from [your comment](https://github.com/freeCodeCamp/guides/issues/2231#issuecomment-339670898) ? You can commit directly on my branch.

**Comment by HKuz** [comment]
LGTM, @davcri - I added a couple sections (from that comment) and made some text edits on top of your PR. I'm going to merge this to get it in the repo, then any other changes/edits we can get in future PRs! ✅

**Comment by davcri** [comment]
Great! Thank you for your improvements 😄

**Pull request merged**